x/attribute/types/events.go

```diff
@@ -36,10 +42,31 @@ func NewEventAttributeAdd(attribute Attribute, owner string) *EventAttributeAdd
 	}
 }
 
+func NewEventAttributeUpdate(originalAttribute Attribute, updateAttribute Attribute, owner string) *EventAttributeUpdate {
+	return &EventAttributeUpdate{
+		Name:          originalAttribute.Name,
+		OriginalValue: string(originalAttribute.GetValue()),
+		OriginalType:  originalAttribute.AttributeType.String(),
+		UpdateValue:   string(updateAttribute.GetValue()),
```

Comment:
Probably want to base64 encode these rather than using `string(...)`.

Reply:
Not sure if I agree here.  I'm just string what ever value is stored in the kv store, they might already be stored as a base64 encoded, like the attribute type bytes.  We also have the `add` doing the same thing, so I just followed that.  Unless, that should also be doing encoding...

Reply:
I don't think we should be using string context here at all.  The attribute value on chain is `[]byte` array therefor these should be encoded as `base64` strings of that value.

Reply:
I changed them to be base64 encoded.  @dwedul-figure I was thinking how the proto was handled in the CLI too.  I too believe it should be handled like the AttributeType_bytes.